test/JDBC/expected/BABEL-937-vu-prepare.out

```diff
@@ -0,0 +1,107 @@
+create view babel_937_not_append_kp as
```

Comment:
From reviewer's perspective, it is not easy to know what `not_append_kp` means and what's the purpose of each test cases. Not only for this specific case but also for other test cases, please add some comments explaining which functionality is tested.